packages/table/src/regions.ts

```diff
@@ -467,6 +467,16 @@ export class Regions {
         return true;
     }
 
+    public static isRegionValidForTable(region: IRegion, numRows: number, numCols: number) {
+        if (region.rows != null && (region.rows[0] >= numRows || region.rows[1] >= numRows)) {
```

Comment:
i'm confused... there's nothing to remove. @cmslewis was asking if we should _also_ verify `< 0`, not just `>= numRows` as you have here.